bin/mantle-dynamics-L0/4_Rayleigh_Nusselt_Relationship.ipynb

```diff
@@ -0,0 +1 @@
aterial, approximated using the reference density, $\\rho_0$, the thermal expansivity of the material, $\\alpha$, and temprature difference of the material with the surroundings, ${\\Delta}T$. The length scale, thermal conductivity, and the viscosity of the convecting fluid is $D$ , $k$, and $\\eta$, respectively.\n","\n","Therefore, the Rayleigh number physically represents the ratio of the convective to the conductive vigor. In practice, the value of Ra can be understood by the following example scenarios:\n","\n","Consider that:\n","- A cubic millimeter of honey will not convect for a temperature difference of 100K\n","- But a cubic kilometer of honey may\n","\n","Also:\n","- A cubic kilometer of honey will not convect for a temperature difference of 0.1K\n","- But it may with a temperature difference of 100K\n","\n","In summary,\n","- Larger Rayleigh number -> More Vigorous convection -> More heat is transported convectively\n","\n","### Nusselt Number\n","\n","\"Nusselt Number\" represents the ratio of the total heat transfer to the conductive heat tranfer through a boundary.The Rayleigh number has a known empirical relationship with the Nusselt number, which is: $Nu \\approx Ra^\\frac{1}{3} $"]},{"cell_type":"markdown","id":"8749ccea","metadata":{"id":"8749ccea"},"source":["### Exercise: Conduction vs Convection\n","To understand the predominace of convective vs. conductive heat transport, let's modify the Ra number by playing with one of the parameters (viscosity)!\n","This can be done in the *subsection Material model* of the input .prm file, by modifying the viscosity value.\n","\n","For example, halving the default value :\n","\n","`set Viscosity = 2.5e24`\n","\n","Other default material properties to play with in the input file are:\n","\n","  * $\\rho_{0}$ = 3300,\n","  * g  = 9.81,\n","  * $\\alpha$ = $2*10^{-5}$\n","  * $\\Delta T$ = (3600-273) = 3327,\n","  * D = $3*10^{6}$\n","  * $k$ = 4.7\n","  * $c_{p}$ = 1250"]},{"cell_type":"markdown","source":["<figure>\n","<img src=\"images/exercise.png\" width=\"600\"/>\n","</figure>"],"metadata":{"id":"re1xnzwFNjLF"},"id":"re1xnzwFNjLF"},{"cell_type":"markdown","source":[" &nbsp;<div style=\"text-align: right\">  \n","    &rarr; <b>NEXT: [Different tectonic regimes](./5_different_tectonic_regimes.ipynb) </b> <a href=\"\"></a> &nbsp;&nbsp;\n","     <img src=\"../assets/education-gem-notebooks_icon.png\" alt=\"icon\"  style=\"width:4%\">\n","  </div>"],"metadata":{"id":"ArvZpz_5tL2_"},"id":"ArvZpz_5tL2_"},{"cell_type":"code","source":[],"metadata":{"id":"Ny9fCBhwtVKv"},"id":"Ny9fCBhwtVKv","execution_count":null,"outputs":[]}],"metadata":{"kernelspec":{"display_name":"Python 3 (ipykernel)","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.5"},"colab":{"provenance":[]}},"nbformat":4,"nbformat_minor":5}
```

Comment:
<p>vs -&gt; vs.</p><p> </p><p>sometimes we refer to it as .prm and other times prm (without the dot). We should stick to one or the other. My preference is .prm.</p><p> </p><p>For example, halving the default value :  -&gt; For example, halve the default value:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>